docs/src/main/sphinx/functions/aggregate.md

```diff
@@ -241,28 +241,33 @@ results in:
  200 | b,c
 ```
 
-This aggregation function can be also used with the `FILTER` keyword to specify
-which rows are processed during the `listagg` aggregation:
+This aggregation function supports 
+{ref}`filtering during aggregation <aggregate-function-filtering-during-aggregation>`:
 
-```sql
-SELECT listagg(value, ',')
-    WITHIN GROUP (ORDER BY id)
-    FILTER (WHERE id % 2 = 0) csv_value
-FROM (VALUES
+```
+SELECT 
+    id,
+    listagg(value, ',')
+        WITHIN GROUP (ORDER BY id)
+        FILTER (WHERE id % 2 = 0) csv_value
+FROM (VALUES 
     (1, 'a'),
     (2, 'b'),
     (3, 'c'),
-    (4, 'd')
+    (2, 'd')
```

Comment:
Should we rename the id column since it is no longer unique?

Reply:
Slightly changed the sample query to give it more color.